src/pages/Admin/EntranceExperience/EntranceExperience.tsx

```diff
@@ -128,6 +128,7 @@ const EntranceExperience: React.FC<EntranceExperienceProps> = ({
               register={register}
               fieldName="entrance"
               showTitle={false}
+              editing={venue.entrance}
```

Comment:
Should this also have an `errors` key set like the other examples here?

Reply:
Good point. I'll add those too.